src/System.Diagnostics.Process/tests/System.Diagnostics.Process.Tests/ProcessTest.cs

```diff
@@ -13,7 +13,7 @@ namespace System.Diagnostics.ProcessTests
     public partial class ProcessTest : IDisposable
     {
         private const int WaitInMS = 100 * 1000; 
-        private const string CoreRunName = "corerun";
+        private const string CoreRunName = "corerun.exe";
```

Comment:
Why this change?  This will break on Unix, where corerun doesn't have this suffix.